zerver/tests/test_signup.py

```diff
@@ -2788,7 +2788,7 @@ def test_realm_creation_through_ldap(self) -> None:
                                                    from_confirmation='1',
                                                    # Pass HTTP_HOST for the target subdomain
                                                    HTTP_HOST=subdomain + ".testserver")
-            self.assert_in_success_response(["We just need you to do one last thing.",
+            self.assert_in_success_response(["You're almost there.",
```

Comment:
These changes seem to be accidentally reverting part of 536de666df309f32e66aeb2ca05c2a7152acfe5b and are likely in error.